Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM python:3.11-slim
+FROM python:3.13-slim
 
-WORKDIR /code
+WORKDIR /app
 
-COPY ./requirements.txt /code/requirements.txt
+COPY ./requirements.txt /app/requirements.txt
```

Comment:
Sorry to nitpick, but let's put requirements in the root - so `/requirements.txt`, and let's change pip install accordingly.

It's just so we know that everything that is in /app mirrors out local app folder.